src/Silk.Core.Logic/Startup.cs

```diff
@@ -54,10 +71,10 @@ private static void AddLogging(IHostBuilder host)
             host.ConfigureLogging((builder, _) =>
                 {
                     var logger = new LoggerConfiguration()
-                        .WriteTo.Console(outputTemplate: LogFormat, theme: SerilogThemes.Bot)
-                        .WriteTo.File("./logs/silkLog.log", LogEventLevel.Verbose, LogFormat, rollingInterval: RollingInterval.Day, retainedFileCountLimit: null)
-                        .MinimumLevel.Override("Microsoft", LogEventLevel.Error)
-                        .MinimumLevel.Override("DSharpPlus", LogEventLevel.Warning);
+                        .WriteTo.Console(outputTemplate: StringConstants.LogFormat, theme: SerilogThemes.Bot)
+                        .WriteTo.File("./logs/silkLog.log", LogEventLevel.Verbose, StringConstants.LogFormat, rollingInterval: RollingInterval.Day, retainedFileCountLimit: null)
+                        .MinimumLevel.Override("Microsoft", LogEventLevel.Warning)
+                        .MinimumLevel.Override("DSharpPlus", LogEventLevel.Fatal);
```

Comment:
TIL - Something both you and I might consider, a log database with an overlaying DB instead of log files.